### ISSUE TITLE
Lockscreen wallpaper?

### ISSUE THREAD

**Issue by flleeppyy** [cvzi/WallpaperExport#13]
I'd like to export my lockscreen wallpaper but I'm not seeing anything for that in the app.

**Comment by cvzi** [comment]
I will look into it.

Which Android version do you use?

You could try one of the older versions (1.0.0 or 1.0.2), maybe one of them works:
https://f-droid.org/packages/com.github.cvzi.wallpaperexport/#latest

**Comment by cvzi** [comment]
I found the problem and I think I solved it.

Update is here: https://github.com/cvzi/WallpaperExport/releases/tag/v1.1.0

F-Droid will take a few days to pick up the new version

**Comment by flleeppyy** [comment]
thank you!

**Comment by zifariux** [comment]
hello. I'm also having the same problem, I tried using all versions and non of them show my lockscreen, i'm on android 11 and it only show built in wallpaper and system wallpaper

**Comment by cvzi** [comment]
I don't know what could be the problem. I checked Android 11 on the emulator and it works. I don't have an actual phone with Android 11 to double check it.

**Comment by zifariux** [comment]
i asked my bro to install it on his phone, we have the same phone, redmi note 8 pro, the app shows the lockscreen option although it only shows a white image

**Comment by cvzi** [comment]
Thanks. It is strange that it fails work for both, but different problem. I'll look into it. I think I know someone who has a phone with MIUI operating system, so it should be similar to Redmi, maybe I can test it on their phone.

**Comment by cvzi** [comment]
I checked on a Redmi Android 11 phone with MIUI operating system and it doesn't show the lockscreen wallpaper for me either. I searched Google and it seems that MIUI uses a custom software for lockscreen wallpapers. So unfortunately it won't be possible to export the lockscreen on MIUI. 
(maybe it is possible with root access, but that is not something I have experience with)

(It worked on a Huawei phone with Android 11 no MIUI)
